elfi/client.py

```diff
@@ -70,37 +71,79 @@ def total(self):
 
     @property
     def num_ready(self):
-        return self.total - len(self.pending_indices)
+        return self.total - self.num_pending
```

Comment:
:D I guess my initial thought was that `n_something` would refer to e.g. sample sizes or quantities that you typically would use in formulas, but `num_something` to other things. But this can be changed if needed.